.github/workflows/bat.yml

```diff
@@ -26,10 +26,10 @@ jobs:
       fail-fast: true
       matrix:
         include:
-          - os: ubuntu-latest
+          - os: ubuntu-20.04
             release: latest
             command: assert(~isempty(regexp(version('-release'), '\d{4}.')))
-          - os: ubuntu-18.04
+          - os: ubuntu-22.04
```

Comment:
Maybe keep the top one latest and and move the R2020a one up to 20.04? Latest is 20.04 at the moment but it will help us make sure the action is working with whatever latest happens to be in the future.